libs/community/tests/integration_tests/chat_models/test_litellm_router.py

```diff
@@ -43,6 +56,39 @@
 ]
 
 
+# from https://stackoverflow.com/a/78573267
+def aiter_to_iter(it: AsyncIterator) -> Generator:
```

Comment:
Can we implement a synchronous `_get_fake_results_generator` instead of implementing this conversion?